src/libraries/System.Private.CoreLib/src/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,8 @@
+<linker>
```

Comment:
It would then need to be duplicated, and we'd get into the same situation we have with the ILLinkTrim.xml file, right? Just like we have with #37255

My thinking was that everything defined in here so far is "common" to both CoreCLR and Mono, so it should go in `src\libraries`.

And once we have specific feature switches between the two, we would have the specific pieces in the coreclr or mono locations and then have a build step that would combine `shared` + `Mono` together.

Reply:
I agree that it'd be great to avoid the duplication. However, we know already that we'll need for Mono SPC's at least 10 different versions of ILLink.Substitutions.xml and some stuff will be common but the files won't be identical. Perhaps we could write a simple task which would combine ILLink.Substitutions.xml from different locations together.

Reply:
I have added build support to combine multiple substitution files during the here. I needed to add a new Task, so I am doing it with https://github.com/mono/linker/pull/1234. Once that goes in, I can split the build infrastructure changes here into a separate PR to get it in faster.